Sources/Deprecations+Removals.swift

```diff
@@ -1,6 +1,12 @@
 import Foundation
 import Dispatch
 
+extension FlattenStrategy {
+	/// Renamed to `dropOnBackpressure`.
+	@available(*, deprecated, renamed:"dropOnBackpressure")
```

Comment:
Not sure how strict we are on space after colon, but I just noticed this one.

Reply:
We've been ad hocly following Swift stdlib style, and there is no space after colon in this context.